src/gm3/components/layers/wms.js

```diff
@@ -46,7 +46,10 @@ function defineSource(mapSource) {
         url: mapSource.urls[0],
         // This is a carry over from previous generations behaviour.
         ratio: 1.0,
-        params: Object.assign({'LAYERS': layers.join(',')}, mapSource.params),
+        params: Object.assign({
+            'VERSION': '1.1.0',
```

Comment:
Version 1.1.0 looks odd.  I've seen 1.1.1 and 1.3.0 in the wild.  It looks like OL4 wants to use 1.3.0 by default.

http://openlayers.org/en/latest/apidoc/ol.source.ImageWMS.html